### ISSUE TITLE
Website: Next button text is cut off

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom-website#1134]
## Detailed description

> Describe in detail the issue you're having.

Mike pointed out the next button on Carbon for IBM.com is having some issues: 
- text within next button is cut off
- possibly has extra padding on the top (could cause cutting off text?)

Seeing mostly on the homepage, but also on the Dev Get started page

**Carbon for IBM.com**
| Carbon for IBM.com | Carbon |
| --- | --- |
| <img width="410" alt="Screen Shot 2021-08-11 at 9 01 26 AM" src="https://user-images.githubusercontent.com/45692622/129033346-efa83a90-8731-48fc-92c3-e67afeaf2e65.png"> | <img width="411" alt="Screen Shot 2021-08-11 at 9 01 59 AM" src="https://user-images.githubusercontent.com/45692622/129033608-cd1aaa1b-8ee7-4a92-8a81-c88a2c1d80b8.png"> |

**Possible extra padding**
<img width="413" alt="Screen Shot 2021-08-11 at 9 01 38 AM" src="https://user-images.githubusercontent.com/45692622/129033349-029946dc-8876-4336-a8d5-9cd42822ec30.png">

**Dev get started** 
<img width="419" alt="Screen Shot 2021-08-11 at 9 08 49 AM" src="https://user-images.githubusercontent.com/45692622/129034395-1133c1f9-6fef-4e99-ba5a-f2f6b40089cf.png">

> Is this a feature request (new component, new icon), a bug, or a general
> issue?

bug

> Is this issue related to a specific component?

Next button on Carbon for iBM.com (gatsby site)

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?

Screen shots are inspector on Chrome, but can reproduce on Safari on iPhone 12 pro

**Comment by janchild** [comment]
Had discussion with @shixiedesign about creating new page titles:
- `What is Carbon for IBM.com?` changes to `What we do`
- `Get started` changes to `Steps to adopt`

These shorter page titles should fix the issue. If necessary, I can change `About Carbon for IBM.com` to `About us`. 

And I'll do a very quick pass to clean up the content.

**Comment by janchild** [comment]
[Link to PR](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/1138) and description of work included.

The PR shortens the pages titles for the two pages within `About Carbon for IBM.com` to prevent the breakage as shown in the above issue. 

Page name changes:

`What is Carbon for IBM.com` was changed to `What we provide`
`Get started` was changed to `Steps to adopt`

I also did a light clean up pass of the content:

- cleaning up the three paragraph page description
- adding an overview to provide some structure
- reducing the length of headings, 
- reducing the length of paragraphs to facilitate scanning, and 
- overall, smoothing content to create more of a flow on each page. 

These pages look better but they still need a thorough review of the content.
